src/handlers/net/NetStreamHandler.h

```diff
@@ -86,6 +88,8 @@ class NetworkMetricsBucket final : public visor::AbstractMetricsBucket
             , TCP_SYN("packets", {"protocol", "tcp", "syn"}, "Count of TCP SYN packets")
             , total_in("packets", {"in"}, "Count of total ingress packets")
             , total_out("packets", {"out"}, "Count of total egress packets")
+            , total_unk("packets", {"unknown"}, "Count of total unknown direction packets")
```

Comment:
should it be `unknown` or` unknown_dir` ?

Reply:
Yeah, I debated that myself. When viewed in context with `packets_in` and `packets_out` metrics it kinda makes sense, but when viewed in isolation it's really not clear what `packets_unknown` means. That's why I suggested `packets_unknown_direction` (or `packets_unknown_dir` to make it shorter).

Reply:
done